src/_footer.scss

```diff
@@ -18,24 +18,6 @@
   display: grid;
 }
 
-/* Override the default settings */
-.ccpa-show-settings {
-  &:hover {
-    color: #004972 !important;
-    text-decoration: underline !important;
-  }
-  &:focus::before {
-    border: none !important;
-  }
-  background-color: transparent !important;
-  border: none !important;
-  font-family: 'Inter', 'Helvetica Neue', Arial, sans-serif !important;
-  font-size: 1.125rem !important;
-  font-weight: 400 !important;
-  color: #00688d !important;
-  padding: 0 !important;
-}
```

Comment:
This looks to be copy/pasted from Figma, which is not recommended. Removing in favor of default styles from the Paragon design system.